src/textual/widgets/_text_log.py

```diff
@@ -43,8 +43,9 @@ def __init__(
         name: str | None = None,
         id: str | None = None,
         classes: str | None = None,
+        disabled: bool = False,
```

Comment:
Ditto here about the docstring.

Reply:
Same as with `Tree` here. I'm loathe to turn a PR about adding `disabled` into a general "add more docs" PR too. Once this one is in I think it'll make sense to do another "hunt down missing docstrings" roundup.